app/scripts/projects/module.ts

```diff
@@ -179,7 +179,7 @@ module ngApp.projects {
         ): ng.IPromise => {
           return $http({
             method: 'GET',
-            url: `${config.api}/analysis/survival?filters={"op":"=","content":{"field":"cases.project.project_id","value":"${$stateParams["projectId"]}"}}`,
+            url: `${config.api}/analysis/survival?filters=[{"op":"=","content":{"field":"project.project_id","value":"${$stateParams["projectId"]}"}}]`,
```

Comment:
can you add `cases` back? it makes it so the filters can be reused across different types - doesn't matter here since it is against case but for consistency